contents/handbook/people/spending-money.md

```diff
@@ -50,18 +50,20 @@ PostHog will provide you with office equipment. Please note that it remains Post
 
 ### Laptop, Monitor, and Accessories
 
-We'd prefer you to use a laptop. This is so when we host meetups in real life, you can easily bring your work with you.
+We'd prefer you to use a laptop. This is so when we host meetups in real life, you can easily bring your work with you. 
 
-Our general laptop of choice is a MacBook Pro. Please look for a recent refurbished model. We'd prefer everyone uses Apple laptops, just to keep life simpler - for example, that means everyone can use the same software, and as we get bigger, it'll mean we're dealing with one supplier, not many.
+Our general laptop of choice is a MacBook Pro. Please look for something with an M1 chip or a recent refurbished model. We'd prefer everyone uses Apple laptops, just to keep life simpler - for example, that means everyone can use the same software, and as we get bigger, it'll mean we're dealing with one supplier, not many.
 
-The main thing to look for is that you have enough RAM. 8GB is (frustratingly!) the minimum you'll realistically need if you're browsing the web, writing code, running Excel, using Slack etc.
+The main thing to look for is that you have enough RAM. We suggest 16GB if you are buying a Macbook with an M1 chip, or 32GB for a refurbished model. This is (frustratingly!) the minimum you'll realistically need if you're browsing the web, writing code, running Excel, using Slack etc.
```

Comment:
I think for tech folks a macbook pro m1 with 16GB/32GB makes sense, otherwise I actually think 8GB should be fine with the M1 chip. paging @mariusandra as he has one

Reply:
@timgl @mariusandra based on both your sets of comments, shall we say something along the lines of:

- If you are in a technical or design role, we recommend a Macbook Pro with an Apple Silicon processor and at least 16GB of RAM. 
- If you are in a non-technical role, we recommend a Macbook Air with an Apple Silicon processor and 8GB of RAM. 
- These are guidelines - the most important thing is that you select the model that is appropriate for your needs. 

(And no reference to refurbished laptops any more)

Reply:
Good for me in general, especially the third point about these being just guidelines.

A little bit of extra context on the dev side:
- Apple Silicon chips max out at 16GB right now. For development, especially if you're using a lot of virtualised environments (docker) or doing heavy Clickhouse queries, 32+GB is still better.
- There are still some compatibility issues with Apple Silicon. For example James G probably can't use it as his daily driver due to the nature of his work (a lot of server/orchestration stuff). This will get fixed over the next months, so hopefully when the next hires start, it'll be a non-issue.

I still use my Intel-based mac with 64GB of RAM for daily work and the new Air in the evenings on the couch for lighter tasks because of the software issues. However as all macs are moving to Apple Silicon, we should still avoid new Intel-based macs when we can.

Reply:
Made some tweaks as discussed (and tidied the layout a bit as well)